sysbuild/CMakeLists.txt

```diff
@@ -268,6 +268,13 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_pre_cmake)
         set_config_bool(${DEFAULT_IMAGE} CONFIG_HAVE_CUSTOM_LINKER_SCRIPT y)
         set_config_bool(${DEFAULT_IMAGE} CONFIG_BUILD_NO_GAP_FILL y)
 
+        # Set required options if MCUmgr img mgmt is used
+        if(SB_CONFIG_MCUBOOT_MODE_DIRECT_XIP)
```

Comment:
what about `SB_CONFIG_MCUBOOT_MODE_DIRECT_XIP_WITH_REVERT` config? should it be also checked here?

Reply:
DirectXIP with revert mode is not supported in QSPI split image mode because if you confirm one and do not confirm the other, you will reboot into a bricked module